Report/Template.Rmd

```diff
@@ -243,17 +251,267 @@ datasets = read.and.clean(dataset = "easySHARE_rel6_0_0.rda", wav = 99)
+
+library("knitr")
+knitr::kable(t(round(sum.stats.tot, 2)), caption = "Summary Statistics over all groups")
+
+
+knitr::kable(labor.part.m, caption = "Men: Labor Force Participation Rate by age")
+knitr::kable(labor.part.f, caption = "Women: Labor Force Participation Rate by age")
+
+```
+
+
+Having obtained the share of labor force participation grouped by gender, country and age as well as the according health variables, striking values can be presented.
+
+Among the selected countries, France, Greece and Belgium have the fewest people among the oldest age group, which is ranging from 60 to 64 years. In the middle age group, ranging from 54-59, the German population is sparsely represented. Within the youngest age group, including people from 50 to 54 years old, Italy and Austria have the lowest share.  
+Instead, Austria represents by far the largest share in the oldest age group. Belgium, Italy, the Netherlands and Sweden dominate the highest ranks in the group ranging from 54 to 59 years. Additionally, Greece solely leads the share in the youngest age group. 
+
+Concerning health statistics, Switzerland and the Scandinavian countries stand out positively. Together, they from a group that has an overall low percentage of obese people while achieving high shares of self-reported good health. Among all other countries, the Swiss have the least number of chronic diseases and fewest overweight. Only the Greek have lower share of ADL's. 
+
+Switzerland and the Nordic countries additionally present the least non-participation rates and the highest full-time working rates. 
+Summary Statistics show that those countries, which strive in health topics are related to high labor force participation rates in all categories. Despite this, the only candidate for multicollinearity is the age dummy in the age group 50, which is omitted before model calculation. 
+
+Generally, labor force participation is higher for males than for females. This applies to part-time work, too.
+
+The overall results of the descriptive statistics align well with @kalwij2005labour
+
+\newpage
+
 
 
 \section{Crosssectional probit regression}
-JULIAN
-\subsection{Theory and Design}
-\subsection{Implementation}
-\subsection{Empirical Results}
+
+## Theory and Design 
+
+Probit models allow for the estimation of binary, random response variables.  
+Probit and Logit models mitigate problems that would occur in case a (linear) regression would be used for a similar task. In this case, the range of the response variable would not be restricted to the interval (0,1). Consequently, as probabilities are commonly defined on this space, non-interpretable probability estimations would be delivered.
+The estimated coefficients are subject to a Wald Test on joint significance. 
+
+The model formula holds under the assumption that $\epsilon \sim N(0,1)$
+
+
+$$
+P(Y = 1 | X = x) = \Phi (x \beta)'
```

Comment:
Please review formula - I think its not correct (tilde in wrong place), please use a source